creator-node/src/services/stateMachineManager/index.js

```diff
@@ -27,11 +29,28 @@ class StateMachineManager {
       logger.error(`updateEndpointToSpIdMap Error: ${e.message}`)
     }
 
-    await this.stateMonitoringQueue.init(audiusLibs)
-  }
+    // Initialize queues
+    const stateMonitoringQueue = new StateMonitoringQueue()
+    const stateReconciliationQueue = new StateReconciliationQueue()
+    const stateMonitoringQueueQueue = await stateMonitoringQueue.init({
```

Comment:
wait lmao wat
literally QQ

why do this instead of
```
let queue = new Queue()
queue = await queue.init()
```
or
```
const queue = await (new Queue()).init()
```

Reply:
hahaha good point

for now I'm adding a change where we actually need the StateMonitoringQueue class and its BullQueue so I think it's best to leave this and to later rename the StateMonitoringQueue class to something like StateMonitoringManager or StateMonitoringQueueManager (same for StateReconciliationQueue)

Reply:
sg!